gtsam/slam/dataset.h

```diff
@@ -236,13 +243,23 @@ struct SfmTrack {
   Point3 point3() const {
     return p;
   }
+  void add_measurement(pair<size_t, gtsam::Point2> m) {
+    Measurements.push_back(m);
+  }
+
+  SfmMeasurements& measurements() {
```

Comment:
Could you point me to where this method is being used? I'm not sure why it is returned by reference.

Reply:
Nowhere, actually. It was originally intended to return all measurements in a track(I wanted to avoid using a for loop to get measurements, but it returns an SfmMeasurement object, which we'd need to wrap to get the elements of, so I decided not to use it)